web/static/help/Messaging_es.md

```diff
@@ -0,0 +1 @@
+../../../doc/help/Messaging_es.md
```

Comment:
oops...wait.  anyting in docs isn't included in the build and is being moved to the mattermost/docs repo.  The files in web/static/help/ are help included with the app.  I think the one in web/static/help/Messaging_en.md needs to have the text and not be a relative link.